pkgs/by-name/nm/nmh/package.nix

```
@@ -90,6 +92,7 @@ stdenv.mkDerivation (finalAttrs: {
       claws-mail's mail folders.  Most other mail clients have migrated to
       maildir.
     '';
+    maintainers = with lib.mainainters; [ normalcea ];
```

This is what's causing the build to fail.

```suggestion
    maintainers = with lib.maintainers; [ normalcea ];
```